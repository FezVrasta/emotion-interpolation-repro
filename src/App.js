// @flow
import styled from '@emotion/styled';
import withFallback from './withFallback';
import textStyles from './textStyles';

const App = styled.div`
  color: ${withFallback(props => props.theme.primary)};
  ${textStyles('foo')};
`;

export default App;
