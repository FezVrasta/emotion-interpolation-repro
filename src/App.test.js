// @flow
import React from 'react';
import { mount } from 'enzyme';
import App from './App';

it('it does not break', () => {
  expect(mount(<App />)).toMatchSnapshot();
});
