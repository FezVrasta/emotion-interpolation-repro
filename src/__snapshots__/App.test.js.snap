// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it does not break 1`] = `
.emotion-0 {
  color: red;
}

<Styled(div)>
  <div
    className="emotion-0"
  />
</Styled(div)>
`;
